src/Controller/NoteController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\CaseEntity;
+use App\Entity\Note;
+use App\Entity\User;
+use App\Form\NoteType;
+use App\Repository\NoteRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Knp\Component\Pager\PaginatorInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Entity;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/case")
+ */
+class NoteController extends AbstractController
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
+    /**
+     * @Route("/{id}/notes", name="note_index", methods={"GET", "POST"})
```

Comment:
Consider moving this part of the route `/{id}/notes` to the route annotation above the class as its used multiple times throughout this controller.